README.md

```diff
@@ -36,7 +36,7 @@ If you have a Node and Go environment you can also build the production version
 
 ```
 npm install
-NODE_ENV=production webpack -p --progress
+npm run-script build
```

Comment:
Anyway it is better than to ask user to have webpack installed in the system as this way we control version and configuration.